cmake/radium_setup_functions.cmake

```diff
@@ -203,7 +226,7 @@ function(configure_bundled_Radium_app)
             foreach( rsc \${instRsc})
                 file(COPY \${rsc} DESTINATION ${CMAKE_INSTALL_PREFIX}/bin/${ARGS_NAME}.app/Contents/Resources)
             endforeach()
-            fixup_bundle(${CMAKE_INSTALL_PREFIX}/bin/${ARGS_NAME}.app \"\" \"${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR};${CMAKE_INSTALL_PREFIX}/lib\")
```

Comment:
Why removing `${CMAKE_INSTALL_PREFIX}/lib` ?

Reply:
because it is the same as `${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}`

Reply:
ok